Wind direction changed to abbreviated Cardinal values

nikhiltaborda opened the pull request:
- Added new function getDescriptiveWindDirectionAbbreviation() to Weather class for abbreviated cardinal values.

nikhiltaborda committed:
Wind direction changed to abbreviated Cardinal values

- Added new function getDescriptiveWindDirectionAbbreviation() to Weather class for abbreviated cardinal values.

blubs reviewed (changes requested):
Client requested wind directions to include the 16 cardinal & inter-cardinal directions, too.

nikhiltaborda committed:
Wind direction changed to abbreviated Cardinal values

- Added new function getDescriptiveWindDirectionAbbreviation() to Weather class for abbreviated cardinal values.
- Total 16 cardinal directions added.

blubs reviewed (approved):
This is without a single doubt the most beautiful code I have ever seen.
